src/components/description_list/_description_list.scss

```
@@ -173,11 +173,10 @@
     .euiDescriptionList__title {
       @include euiFontSizeS;
       display: inline;
-      border-radius: $euiBorderRadius;
-      font-weight: $euiFontWeightRegular;
-      background: $euiColorLightestShade;
-      border: $euiBorderThin;
-      padding:  0 $euiSizeXS;
+      border-radius: $euiBorderRadius / 2;
```

One last suggestion that I just caught. This variable will work for both themes and result as `4px` and `2px`.

```suggestion
      border-radius: $euiBorderRadiusSmall;
```